docs/release-info/release-notes.md

```diff
@@ -57,50 +57,609 @@ For tips about how to write a good release note, see [Release notes](https://git
+
+![Load query detail archive](./assets/image02.png)
+
+[#15632](https://github.com/apache/druid/pull/15632)
+
+#### Improved lookup dialog
+
+The lookup dialog in the web console now includes following optional fields. See [JDBC lookup](https://druid.apache.org/docs/latest/development/extensions-core/lookups-cached-global#jdbc-lookup) for more information.
+
+* Jitter seconds
+* Load timeout seconds
+* Max heap percentage
+
+![Lookup dialog](./assets/image01.png)
+
+[#15472](https://github.com/apache/druid/pull/15472/)
+
+#### Improved time chart brush and added auto-granularity
+
+Improved the web console **Explore** view as follows:
+
+* Added the notion of timezone in the explore view.
+* Time chart is now able to automatically pick a granularity if "auto" is selected (which is the default) based on the current time filter extent.
+* Brush is now automatically enabled in the time chart.
+* Brush interval snaps to the selected time granularity.
+* Added a highlight bubble to all visualizations (except table because it has its own).
+
+[#14990](https://github.com/apache/druid/pull/14990)
+
 #### Other web console improvements
 
-### Ingestion
+* Added the ability to detect multiple `EXPLAIN PLAN` queries in the workbench and run them individually [#15570](https://github.com/apache/druid/pull/15570)
+* Added the ability to sort a segment table on start and end when grouping by interval [#15720](https://github.com/apache/druid/pull/15720)
+* Improved the time shift for compare logic in the web console to include literals [#15433](https://github.com/apache/druid/pull/15433)
+* Improved robustness of time shifting in tables in Explore view [#15359](https://github.com/apache/druid/pull/15359)
+* Improved ingesting data using the web console [#15339](https://github.com/apache/druid/pull/15339)
+* Improved management proxy detection [#15453](https://github.com/apache/druid/pull/15453)
+* Fixed rendering on a disabled worker [#15712](https://github.com/apache/druid/pull/15712)
+* Fix an issue where `waitUntilSegmentLoad` would always be set to `true` even if explicitly set to `false` [#15781](https://github.com/apache/druid/pull/15781)
+* Enabled table driven query modification actions to work with slices [#15779](https://github.com/apache/druid/pull/15779)
+
+### General ingestion
+
+#### Added system fields to input sources
+
+Added the option to return system fields when defining an input source. This allows for ingestion of metadata, such as an S3 object's URI.
+
+[#15276](https://github.com/apache/druid/pull/15276)
+
+#### Changed how Druid allocates weekly segments
+
+When the requested granularity is a month or larger but a segment can't be allocated, Druid resorts to day partitioning.
+Unless explicitly specified, Druid skips week-granularity segments for data partitioning because these segments don't align with the end of the month or more coarse-grained intervals.
```

Comment:
I feel for a user this release not does not add value. I think we should mention the old behavior and the new behavior and what should the person do if he want's the old behavior. 
The reasoning of why can go into the PR description. 

cc @kfaraz @AmatyaAvadhanula

Reply:
@cryptoe , there is no way to get back to the old behaviour now.
Old behaviour was that if allocation is not possible for month, we try week next.
In the new behaviour, we will try skip trying week and try day directly.

In the new behaviour, week segments can only be allocated if the chosen partitioning in the append task is WEEK.